src/docs/product/integrations/index.mdx

```diff
@@ -23,7 +23,7 @@ For more details, see the [full Integration Platform documentation](/product/int
 
 | Integration                                                      | API |
 | ---------------------------------------------------------------- | --- |
-| [Vanta](https://help.vanta.com/hc/en-us/articles/13045851267092) | X   |
+| [Vanta](/product/integrations/compliance/vanta/)                 | X   |
```

Comment:
Do you mind moving this section much lower on the page, maybe just above Cloud Monitoring?